### PR TITLE
Front

### PR DESCRIPTION
**Pull request by jetownfeve21**
wip

### CODE REVIEW

**Comment by jetownfeve21** [comment]
@kelson42 @mgautierfr 
What do you think about these changes ?

**Comment by kelson42** [comment]
@jetownfeve21 Would you be able to upload screenshots? Have you tested on both Windows & Linux? What about high pixel density screens https://github.com/kiwix/kiwix-desktop/issues/157 ?

**Comment by jetownfeve21** [comment]
image
image
image

**Comment by jetownfeve21** [comment]
These changes are only to make the app look more like the model

**Comment by kelson42** [comment]
@jetownfeve21 Lets have a look together to this on Monday.

File: resources/css/style.css [hunk 1]
```diff
@@ -19,7 +19,7 @@ TopWidget {
 TopWidget::separator {
     background: none;
     border: none;
-    icon: none;
```
Comment (mgautierfr) [inline resources/css/style.css]: Why remove this ?

Comment (jetownfeve21) [reply resources/css/style.css]: there is this warning
image

Comment (mgautierfr) [reply resources/css/style.css]: Ok, remove it instead of commenting it and we are good with this PR

Comment (jetownfeve21) [reply resources/css/style.css]: Done

**Pull request merged**